test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -1012,32 +944,6 @@ test.describe( 'Links', () => {
 			] );
 		} );
 
-		test( 'should not display text input when initially creating the link', async ( {
```

Comment:
It seems this test still has a purpose and should just change to select some text before creating the link.